.gitignore

```diff
@@ -23,6 +23,7 @@
 hs_err_pid*
 # Java Flight Recorder
 *.jfr
+javac.*.args
```

Comment:
This file should be generated in target directory, which is already excluded, why add it again?

Reply:
I added it so that I can hide the pico modules locally that are not yet ready to merge down.

Reply:
I was talking about `javac.*.args` - that exclusion should not be needed. Which files match it that you need to add it to exclusion?

Reply:
took that out and pushed.